src/mame/namco/namcos21_dsp.cpp

```diff
@@ -65,21 +65,22 @@ void namcos21_dsp_device::device_reset()
 
 uint16_t namcos21_dsp_device::winrun_dspcomram_r(offs_t offset)
 {
-	int bank = 1-(m_winrun_dspcomram_control[0x4/2]&1);
-	uint16_t *mem = &m_winrun_dspcomram[0x1000*bank];
+	unsigned bank = !BIT(m_winrun_dspcomram_control[0x4/2], 0);
+	uint16_t *mem = &m_winrun_dspcomram[0x1000 * bank];
+
 	return mem[offset];
 }
 void namcos21_dsp_device::winrun_dspcomram_w(offs_t offset, uint16_t data, uint16_t mem_mask)
 {
-	int bank = 1-(m_winrun_dspcomram_control[0x4/2]&1);
-	uint16_t *mem = &m_winrun_dspcomram[0x1000*bank];
+	unsigned bank = !BIT(m_winrun_dspcomram_control[0x4/2], 0);
+	uint16_t *mem = &m_winrun_dspcomram[0x1000 * bank];
+
 	COMBINE_DATA( &mem[offset] );
 }
```

Comment:
Implicit `bool` to `int` conversions are bad form – you can use `BIT(~m_winrun_dspcomram_control[0x4/2], 0)` to invert the bit.

Reply:
I would argue that the former is more readable, but addressed either way.